docs/Installing.asciidoc

```diff
@@ -375,14 +375,22 @@ directories, most of which must be owned by the user that runs openQA
 * +images+ is where the server stores test screenshots and thumbnails
 * +share+ contains shared directories for remote workers, can be owned by root
 * +share/factory+ contains test assets and temp directory, can be owned by root but sysadmin must create subdirs
-* +share/factory/iso+ contains ISOs for tests
-* +share/factory/hdd+ contains hard disk images for tests
-* +share/factory/repo+ contains repositories for tests
-* +share/factory/other+ contains miscellaneous test assets (e.g. kernels and initrds)
+* +share/factory/iso+ and +share/factory/iso/fixed+ contain ISOs for tests
+* +share/factory/hdd+ and +share/factory/hdd/fixed+ contain hard disk images for tests
+* +share/factory/repo+ and +share/factory/repo/fixed+ contain repositories for tests
+* +share/factory/other+ and +share/factory/other/fixed+ contain miscellaneous test assets (e.g. kernels and initrds)
```

Comment:
I guess you don't need to mention the subdirectory `fixed` here explictly. The explanation that comes further down is pretty good.